UIFix.md

```diff
@@ -0,0 +1,21 @@
+**Summary**
+
+Fix the ability to move around in TestMode 
```

Comment:
I think this could be clarified. The widget enables the user to change the state of CellProfiler’s test mode (i.e. run test mode from a different position, or, module) rather than “move around.”